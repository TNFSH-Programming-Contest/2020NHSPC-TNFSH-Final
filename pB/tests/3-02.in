85
-158.242 13.507
-154.045 143.603
-75.708 153.395
-86.899 177.176
-149.848 184.17
-254.764 270.901
-295.332 381.412
-349.888 410.789
-326.107 298.878
-425.427 307.272
-394.652 247.12
-314.916 184.17
-321.91 69.462
-479.984 66.665
-477.186 -115.19
-544.332 -120.785
-604.484 -32.656
-572.31 217.743
-537.338 114.226
-376.467 121.221
-386.259 184.17
-498.169 174.378
-551.326 603.834
-489.776 603.834
-491.175 353.435
-376.467 353.435
-407.242 461.148
-396.051 610.828
-253.365 649.997
-159.641 645.8
-169.433 595.44
-196.011 637.407
-274.349 595.44
-293.933 465.345
-239.377 360.429
-176.427 263.906
-159.641 318.463
-221.191 423.378
-221.191 594.042
-140.056 524.098
-103.685 601.036
-61.719 501.716
-18.354 567.463
16.618 526.895
40.8 501.8
107.545 507.311
129.927 444.362
66.978 426.176
2.629 493.322
-142.854 421.98
-71.511 324.058
-15.556 340.845
68.377 374.418
138.32 280.693
96.354 205.153
19.416 254.0
-75.708 234.53
1.23 56.872
54.388 51.277
36.202 186.968
148.113 115.625
148.113 35.889
103.349 75.058
89.36 -4.678
160.703 -38.251
149.511 -113.791
45.995 -57.836
58.584 -130.577
-14.157 -124.982
-2.966 -49.442
-113.478 86.249
-131.663 -116.588
-175.028 -88.611
-207.0 -120.785
-321.91 -138.971
-271.551 -94.206
-201.607 -38.251
-242.174 -21.465
-334.5 -78.819
-429.624 -83.015
-436.618 34.49
-398.849 45.681
-394.652 -27.06
-330.304 -27.06
-330.304 21.9
-57.907 477.433
