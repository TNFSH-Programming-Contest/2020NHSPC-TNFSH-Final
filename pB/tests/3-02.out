1867595.328